core/pom.xml

```diff
@@ -270,7 +270,7 @@
     <dependency>
       <groupId>org.tachyonproject</groupId>
       <artifactId>tachyon-client</artifactId>
-      <version>0.8.1</version>
+      <version>0.8.2</version>
       <exclusions>
         <exclusion>
           <groupId>org.apache.hadoop</groupId>
```

Comment:
Hey, quick question: why do we need these exclusions? AFAIK `tachyon-client` is an uber-JAR which shades all of its dependencies: http://mvnrepository.com/artifact/org.tachyonproject/tachyon-client/0.8.2